File: Alex/wk01/3-wed/arrays.js
//1.

var days_of_the_week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
console.log(days_of_the_week);

//2.

days_of_the_week.unshift(days_of_the_week.pop());
console.log(days_of_the_week);

//3.

var weekdays = days_of_the_week.slice(1,6);
console.log(weekdays);

var weekends = [].concat(days_of_the_week.slice(0,1), days_of_the_week.slice(-1));
console.log(weekends);
weekends.reverse();
console.log(weekends);

//4. ~~~~~~~~~~~

//5.

weekdays.sort();
console.log(weekdays);

File: Alex/wk04/1-mon/arrays.rb
require 'pry'

days_of_the_week = 'Monday Tuesday Wednesday Thursday Friday Saturday Sunday'

part1 = days_of_the_week.split

part2 = part1.unshift(part1.pop)

part3 = Array.new(2)

part3[0] = part2[1, 5]
part3[1] = [part2[6], part2[0]] 

part4 = part3[0]

part5 = part4.sort

binding.pry

puts('end of program')